doc/book/box/stream.rst

```diff
@@ -0,0 +1,91 @@
+..  _box_stream:
+
+Streams and interactive transactions
+====================================
+
+..  _box_stream-overview:
+
+Overview
+--------
+
+Since :tarantool-release:`2.10.0-beta1`, iproto implements streams and interactive transactions.
+
+..  glossary::
+
+    Stream
+        A stream is a concept that supports multiplexing several transactions over one connection.
+        All requests in the stream are executed strictly sequentially.
+        It is a more general approach that allows the implementation
+        of :term:`interactive transactions <interactive transaction>`.
+
+Unlike a thread associated with multitasking and execution within a program,
+a stream occurs between a client and a server.
+It deals with connecting and transferring a protocol over the network.
```

Comment:
now it looks like the protocol itself is transferred. but is it so? maybe it is the _data_ that is transferred via the protocol? I think the sentence needs to be rephrased